src/components/Root/Root.js

```diff
@@ -6,11 +6,11 @@ import createBrowserHistory from 'history/createBrowserHistory';
 import { IntlProvider } from 'react-intl';
 import queryString from 'query-string';
 import { ApolloProvider } from 'react-apollo';
-
```

Comment:
Best to avoid personal-preference changes, like removing blank lines, as part of a more substantial change. It makes the PR look bigger and harder to make sense of. Not a big deal, just something to bear in mind.

Reply:
Sure, will take care of this.